### PR TITLE
incrementExact docs added

### PR DESCRIPTION
**Pull request by isanketsaha**
### Description

added new term entry Java Math Functions: incrementExact()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@isanketsaha, I did a first review and I left a few comments.

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  Tags:
+- 'Functions'
+- 'Methods'
+- 'Arithmetic'
+
+  CatalogContent:
+- 'learn-java'
+---
+
+The **`Math.incrementExact()`** method returns the argument incremented by one or throws exception if the result overflows the specified datatype either long or int.
```
[the remark this answers is not in the document]
Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md]: Changes done.

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+```pseudo
+Math.incrementExact(num)
+```
+
+- The `num` argument is a `double` or `int` value.
+
+## Example
+
+The following example uses `Math.incrementExact()` to increases the input by 1:
```
[the remark this answers is not in the document]
Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md]: Changes done.

**Review by SSwiniarski — APPROVED** [review]
@isanketsaha, Looks good for a second review.

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the datatype.'
+Subjects:
+- 'Computer Science'
+  Tags:
+- 'Functions'
+- 'Methods'
+- 'Arithmetic'
+
+  CatalogContent:
+- 'learn-java'
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:13]:
```suggestion
Title: '.incrementExact()'
Description: 'Returns the argument incremented by one, throwing an exception if the result overflows the data type.'
Subjects:
- 'Computer Science'
  Tags:
- 'Functions'
- 'Methods'
- 'Arithmetic'
  CatalogContent:
- 'learn-java'
```

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+The **`Math.incrementExact()`** method returns the argument incremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified datatype either `long` or `int`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:16]:
```suggestion
The **`Math.incrementExact()`** method returns the argument incremented by one. It throws an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows the specified [data type](https://www.codecademy.com/resources/docs/java/data-types) (`long` or `int`).
```

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Syntax
+
+```pseudo
+Math.incrementExact(num)
+```
+
+- The `num` argument is a `long` or `int` value.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:24]:
```suggestion
The `num` parameter is of type `long` or `int`.
```

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example
+
+The following example uses `Math.incrementExact()` to increase the input by 1:
+
+```java
+public class Main {
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:31]:
```suggestion
// Main.java

public class Main {
```

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+```pseudo
+Math.incrementExact(num)
+```
+
+- The `num` argument is a `long` or `int` value.
+
+## Example
+
+The following example uses `Math.incrementExact()` to increase the input by 1:
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:28]:
```suggestion
The following example uses the `.incrementExact()` method to increase the input by 1:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@isanketsaha Finished with my review and left you some comments/suggestions. 😄

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md (hunk 3 above)
Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:13]: Removing these extra lines will make the content overlap. Added screenshot. Please comment
Screenshot 2022-10-21 at 4 22 55 PM

**Comment by isanketsaha** [comment]
@Dusch4593 , please review.

**Comment by isanketsaha** [comment]
@Dusch4593, can you please comment if I need to work on this or if it is good to be merged?

File: content/java/concepts/math-methods/terms/incrementExact/incrementExact.md (hunk 3 above)
Comment (Dusch4593) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:13]: Not sure where you are previewing the Markdown, but when I preview on GitHub, the metadata renders expected w/ the suggested changes:

Screen Shot 2022-10-24 at 7 41 41 AM

Comment (Dusch4593) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md:13]: You can certainly go off an published entry's meta-data to get a sense of what we're looking for: https://github.com/Codecademy/docs/edit/main/content/java/concepts/math-methods/terms/decrementExact/decrementExact.md

Comment (isanketsaha) [reply content/java/concepts/math-methods/terms/incrementExact/incrementExact.md]: @Dusch4593 , Thank you for guiding me and sharing the screenshot. I have made the necessary changes. can you please take a look.

**Review by Dusch4593 — APPROVED** [review]
@isanketsaha Looks like we're good to merge! 🚀 I made a super small edit in 4f87d9a9a993b91f4833dbcfe109db6321be59e5.

**Comment by Dusch4593** [comment]
Congrats on making your contribution to Codecademy Docs, @isanketsaha! 🚀 🎉 🎊

The live entry can be viewed here: https://www.codecademy.com/resources/docs/javascript/objects

Your profile is currently anonymous on the entry; you'll need to add GitHub account to your Codecademy Profile and log in through GitHub:

Screen Shot 2022-10-26 at 11 35 01 AM

**Pull request merged**